drivers/adc/adc_stm32.c

```diff
@@ -227,7 +236,6 @@ struct adc_stm32_data {
 	const struct device *dev;
 	uint16_t *buffer;
 	uint16_t *repeat_buffer;
-
```

Comment:
We usually try not to include unrelated whitespace changes in commits which change functionality. I would recommend dropping this change or moving it to a separate commit.